### PR TITLE
Fix OU name not displayed in event report

### PR DESCRIPTION
**Pull request by luciano-fiandesio**
- DHIS2-6885
- Altered INSERT query that populates the Event analytics table to
fetch the OU name when the Data Element is type 'ORGANIZATIONALUNIT', instead
of the OU UID
- Added unit tests for JdbcEventAnalyticsTableManager
- Minor refactoring

### CODE REVIEW

**Comment by jason-p-pickering** [comment]
Can one of the admins verify this patch?

File: dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcAnalyticsTableManager.java [hunk 1]
```diff
@@ -121,6 +116,20 @@ public JdbcAnalyticsTableManager( IdentifiableObjectManager idObjectManager,
             databaseInfo, jdbcTemplate );
     }
 
+    private List<AnalyticsTableColumn> DEFAULT_COLS = Lists.newArrayList(
```
Comment (larshelge) [inline dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcAnalyticsTableManager.java]: Perhaps rename "DEFAULT" to "FIXED" (as the others are dynamic)

File: dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcAnalyticsTableManager.java [hunk 2]
```diff
@@ -490,6 +484,11 @@ private String getApprovalJoinClause( Integer year )
         return ConcurrentUtils.getImmediateFuture();
     }
 
+    @Override
+    public List<AnalyticsTableColumn> getDefaultColumns() {
```
Comment (larshelge) [inline dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcAnalyticsTableManager.java]: Code style. Rename to getFixedColumns()

File: dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcEnrollmentAnalyticsTableManager.java [hunk 3]
```diff
@@ -231,4 +207,9 @@ protected void populateTable( AnalyticsTableUpdateParams params, AnalyticsTableP
 
         return filterDimensionColumns( columns );
     }
+
+    @Override
+    public List<AnalyticsTableColumn> getDefaultColumns() {
```
Comment (larshelge) [inline dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcEnrollmentAnalyticsTableManager.java]: Code style

File: dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcEventAnalyticsTableManager.java [hunk 4]
```diff
@@ -361,6 +344,25 @@ protected void populateTable( AnalyticsTableUpdateParams params, AnalyticsTableP
         return filterDimensionColumns( columns );
     }
 
+    private String addClosingParentheses( String selectSql )
```
Comment (larshelge) [inline dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/table/JdbcEventAnalyticsTableManager.java]: Move to util class as static method, add test.

**Pull request merged**